daemon/cmd/daemon_main.go

```diff
@@ -1004,6 +1004,18 @@ func init() {
 	flags.MarkHidden(option.BypassIPAvailabilityUponRestore)
 	option.BindEnv(option.BypassIPAvailabilityUponRestore)
 
+	flags.Bool(option.EnableVTEP, defaults.EnableVTEP, "Enable  VXLAN Tunnel Endpoint (VTEP) support")
```

Comment:
I think we should also indicate beta status here:
```suggestion
	flags.Bool(option.EnableVTEP, defaults.EnableVTEP, "Enable VXLAN Tunnel Endpoint (VTEP) support (beta)")
```

Reply:
done